op-validator/cmd/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum-optimism/optimism/op-validator/pkg/service"
+	"github.com/urfave/cli/v2"
+)
+
+const EnvVarPrefix = "OP_VALIDATOR"
+
+var (
+	GitCommit = ""
+	GitDate   = ""
+	Version   = ""
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Version = Version
+	app.Name = "op-validator"
+	app.Usage = "Optimism Validator Service"
+	app.Description = "CLI to validate Optimism L2 deployments"
+	app.Flags = oplog.CLIFlags(EnvVarPrefix)
+	app.Commands = []*cli.Command{
+		{
+			Name:  "validate",
+			Usage: "Run validation for a specific version",
+			Subcommands: []*cli.Command{
+				{
+					Name:  "v1.8.0",
+					Usage: "Run validation for v1.8.0",
+					Flags: append(service.ValidateFlags, oplog.CLIFlags(EnvVarPrefix)...),
+					Action: func(cliCtx *cli.Context) error {
+						return service.ValidateCmd(cliCtx, "v1.8.0")
+					},
+				},
+				{
+					Name:  "v2.0.0",
+					Usage: "Run validation for v2.0.0",
+					Flags: append(service.ValidateFlags, oplog.CLIFlags(EnvVarPrefix)...),
+					Action: func(cliCtx *cli.Context) error {
```

Comment:
Nit: Can't we just have a slice of supported versions with a potential default version, just check and operate from it?

Hardcoding subcommands on a per-version basis seems a bit awkward considering fundamentally all of them are calling `service.ValidateCmd(cliCtx, <version>)`

Reply:
I can make a slice sure